qcelemental/util/misc.py

```diff
@@ -130,3 +130,173 @@ def unnp(dicary, flat=False, _path=None):
                 else:
                     ndicary[k] = v.tolist()
     return ndicary
+
+
+def _norm(points):
+    """
+    Return the Frobenius norm across axis=-1, NumPy's internal norm is crazy slow (~4x)
+    """
+
+    tmp = np.atleast_2d(points)
+    return np.sqrt(np.einsum("ij,ij->i", tmp, tmp))
```

Comment:
not duplicating https://github.com/MolSSI/QCElemental/blob/81c65d815fbb18b8bb689e4a1c4c2b2c896d5c56/qcelemental/util/misc.py#L9-L18, is it?

Reply:
No, this is just a fast norm equivalent to `np.linalg.norm(arr, axis=1)`. Somewhat meaningless, but something I often do.